src/tests/ftest/pool/bad_exclude.py

```diff
@@ -113,52 +83,45 @@ def test_exclude(self):
         saved_svc = None
         saved_grp = None
         saved_uuid = None
-        pool = None
-        try:
-            # initialize a python pool object then create the underlying
-            # daos storage
-            pool = DaosPool(self.context)
-            pool.create(createmode, createuid, creategid,
-                        createsize, createsetid, None)
+        self.prepare_pool()
 
+        try:
             # trash the the pool service rank list
             if not svc == 'VALID':
                 self.cancel("skipping this test until DAOS-1931 is fixed")
-                saved_svc = RankList(pool.svc.rl_ranks, pool.svc.rl_nr)
-                pool.svc = None
+                saved_svc = RankList(
+                    self.pool.pool.svc.rl_ranks, self.pool.pool.svc.rl_nr)
+                self.pool.pool.svc = None
 
             # trash the pool group value
             if connectset == 'NULLPTR':
-                saved_grp = pool.group
-                pool.group = None
+                saved_grp = self.pool.pool.group
+                self.pool.pool.group = None
 
             # trash the UUID value in various ways
             if excludeuuid == 'NULLPTR':
                 self.cancel("skipping this test until DAOS-1932 is fixed")
-                ctypes.memmove(saved_uuid, pool.uuid, 16)
-                pool.uuid = 0
+                ctypes.memmove(saved_uuid, self.pool.pool.uuid, 16)
+                self.pool.pool.uuid = 0
             if excludeuuid == 'CRAP':
                 self.cancel("skipping this test until DAOS-1932 is fixed")
-                ctypes.memmove(saved_uuid, pool.uuid, 16)
-                pool.uuid[4] = 244
+                ctypes.memmove(saved_uuid, self.pool.pool.uuid, 16)
+                self.pool.pool.uuid[4] = 244
 
-            pool.exclude(targets)
+            self.pool.pool.exclude(targets)
```

Comment:
This is the only line that would possibly raise a DaosApiError and therefore should be the only line in the `try`.